src/languages/javascript.js

```diff
@@ -78,14 +78,21 @@ export default function(hljs) {
     NUMBER,
     hljs.REGEXP_MODE
   ];
-  var PARAMS_CONTAINS = SUBST.contains.concat([
-    // eat recursive parens in sub expressions
-    { begin: /\(/, end: /\)/,
-      contains: ["self"].concat(SUBST.contains, [hljs.C_BLOCK_COMMENT_MODE, hljs.C_LINE_COMMENT_MODE])
-    },
+  var PARAMS_CONTAINS = [].concat([
     hljs.C_BLOCK_COMMENT_MODE,
-    hljs.C_LINE_COMMENT_MODE
-  ]);
+    hljs.C_LINE_COMMENT_MODE,
+    // eat recursive parens in sub expressions
+    {
+      begin: /\(/,
+      end: /\)/,
+      contains: ["self"].concat([
+          hljs.C_BLOCK_COMMENT_MODE,
+          hljs.C_LINE_COMMENT_MODE
+        ],
+        SUBST.contains
+      )
+    }
+  ], SUBST.contains);
   var PARAMS = {
```

Comment:
This is just flipped the order and moving the comment modes to the top of the contains list for both PARAMS_CONTAINS and it's recursive sub expression matcher.